submissions/Conversee12/DOM/script.js

```diff
@@ -0,0 +1,83 @@
+    about: "Bullet for My Valentine are a Welsh metalcore band from Bridgend, formed in 1998. The band is composed of Matthew Tuck (lead vocals, rhythm guitar), Michael Paget (lead guitar, backing vocals), Michael Thomas (drums) and Jamie Mathias (bass guitar). Former members include Nick Crandle and Jason James; both were on bass. They were formed under the name Jeff Killed John and started their music career by covering songs by Metallica and Nirvana. Jeff Killed John recorded six songs which were not released; two of these tracks were reworked later in their career as Bullet for My Valentine. Financial difficulties dictated the name change, which was followed by a change in music direction. In 2002, the band secured a five-album deal with Sony BMG.",
+    myBestSong: 'Best song in my opinion : "Your Betrayal"',
+    image: "img/valentine.jpg"
+  },
+  {
+    title: "Thirty Seconds to Mars",
+    about: "Thirty Seconds to Mars (commonly stylized as 30 Seconds to Mars) is an American rock band from Los Angeles, California, formed in 1998. The band consists of brothers Jared Leto (lead vocals, guitar, bass, keyboards) and Shannon Leto (drums, percussion). During the course of its existence, it has undergone various line-up changes. The band's debut album, 30 Seconds to Mars (2002), was produced by Bob Ezrin and released to positive reviews but only to limited commercial success. The band achieved worldwide fame with the release of its second album A Beautiful Lie (2005), which received multiple certifications all over the world. Its next release, This Is War (2009), showed a dramatic evolution in the band's musical style, as it incorporated experimental music as well as eclectic influences. The recording process of the album was marked by a legal dispute with record label EMI that eventually became the subject of the documentary film Artifact (2012).",
+    myBestSong: 'Best song in my opinion : "A Beautiful Lie"',
+    image: "img/mars.jpg"
+  },
+];
+
+function initButtons(container) {
+  return rockBands.forEach((band) => {
+    const { title } = band;
+    const button = document.createElement("button");
+    button.classList.add("buttons");
+    button.innerText = title;
+    button.addEventListener("click", ({ target }) => {
+      showMain(target);
+    });
+    container.append(button);
+  });
+}
+
+function showMain(target) {
+  const content = rockBands.filter(
+    (band) => target.innerText === band.title
+  );
+  const [{ title, myBestSong, about, image }] = content;
+  const bandTitle = document.querySelector(".section__title");
+  const bandImage = document.querySelector(".section__img");
+  const bandInfo = document.querySelector(".section__text");
+  const bandSong = document.querySelector(".section__song");
+  bandTitle.innerText = title;
+  bandSong.innerText = myBestSong;
+  bandInfo.innerText = about;
+  bandImage.src = image;
+}
+
+const sideBar = document.querySelector(".sideBar");
+initButtons(sideBar);
```

Comment:
You use `sideBar` only inside `initButtons` and `initButtons` call only once. You can move `sideBar` into `initButtons`.